src/Symfony/Component/Mime/Tests/EmailTest.php

```diff
@@ -432,8 +432,7 @@ public function testSymfonySerialize()
             }
         ]
     },
-    "body": null,
-    "message": null
```

Comment:
The reason why `message` disappears here is that PHP does not render uninitialized properties when serializing JSON.

`RawMessage::$message` is initialized in the constructor of `RawMessage`, but its child class `Message` skips the parent constructor call, thus the property is left uninitialized. This is okay in the context of `Message` because that class overrides all operations that would access that property.

If I wanted the `message` property to reappear in JSON, I would need to initialize it with `null` and make it nullable. `RawMessage` however assumes that this property is never `null`.

I think it is correct to not render the `message` property. However, this seems to break at least one test from another package that relies on the `message` property to be present.

Reply:
We should relax the test case in TwigBridge then.